internal/repository/policy.go

```diff
@@ -24,7 +24,7 @@ var (
 // update to the RSL.
 func (r *Repository) PushPolicy(ctx context.Context, remoteName string) error {
```

Comment:
We should think about the interface for Push so we know what ref is being pushed. Perhaps we don't want to push both everytime...

Reply:
Should this be a later PR, or should this be incorporated into this one as a different function in cmd/rsl

Reply:
This is another what-should-the-workflow-be question, so let's see.